tests/run_tests.py

```diff
@@ -126,6 +213,10 @@ def SummarizeResults(self):
       for failure in self.failures:
         print failure
 
+      print '\nSummary of Successes:'
```

Comment:
I'm not sure if we want to add this, printing the successes is going to hide the failures.

Reply:
I'll remove it. I added it before I add the suppression list to see what is passing and now there is no need for it.